### ISSUE TITLE
Can't Use Sigils in the Sigil of Holding

### ISSUE THREAD

**Issue by Utildayael** [WayofTime/BloodMagic#891]
#### Issue Description:

This may be a known issue but was unable to find an open issue tracking it so...

Sigils contained in the sigil of holding don't work. You can select them fine but can't activate them. New to 1.9/1.10 BM so assuming it worked like in 1.7 where you cycle through to the sigil you want and can use it just like holding sigil itself.

The only sigil that works proper in there is the sigil of divination. Tried Green Grove, Fast Miner, Blood Lamp and they all do nothing oddly. If I remove them from the sigil of holding, they work fine.
#### What happens:

see above
#### What you expected to happen:

Sigils to operate from within sigil of holding same functionally as in 1.7.10 BM
#### Steps to reproduce:
1.  place sigils into sigil of holding
2.  attempt to use any sigil other than divination
3.  
   ...

---
#### Affected Versions (Do _not_ use "latest"):
- BloodMagic: 1.9.4-2.0.4-56
- Minecraft: 1.10.2
- Forge: 12.18.1.2065

**Comment by TehNut** [comment]
Are the sigils bound to you?

**Comment by TehNut** [comment]
https://streamable.com/ufcu

**Comment by Utildayael** [comment]
Part of problem. 3 of 4 in sigil were bound to me. I had a fast miner I had tossed in without binding first. 

Seems if one non-bound sigil is in there with all bound sigils (and a bound sigil of holding of course) it makes all of them not work aside from divination. 

Once I bound the miner so ALL sigils were bound, they all started working. 

That said... I made a sigil I hadn't yet. Purposely did not bind it and tossed it into my sigil of holding. All sigils worked properly except the one that hadn't been bound. Soooo... Can't reproduce it seems. 

Fwiw I had following sigils in my sigil of holding when I filed the bug:

Divination - bound
Sigil of Holding - bound
Green Grove - bound
Blood Lamp - bound
Fast Miner - not bound

After binging the fast miner and everything working, I made a ender severance and tossed that in unbound which had no effect on the, now working, other 4 sigils as only ender severance didn't work. Took it out. Bound it. Works fine as expected. 

Sorry about that!
